benchmark/cmd/command/banner.go

```diff
@@ -0,0 +1,58 @@
+// Copied from github.com/pyroscope-io/pyroscope/cmd/command/banner.go
```

Comment:
This code is in the same module so we should be able to reuse it relatively easily. Maybe make these functions public?

Reply:
maybe i misunderstood you, but it's two different modules (`pyroscope-io/pyroscope/benchmark/cmd/command`, and `pyroscope-io/pyroscope/cmd/command`)
correct me if i am wrong, but i think modules under `cmd` are not exported publicly?

so what we can do is to just move that gradient code to `pyroscope-io/pyroscope/pkg/cli`, doing it now.